tracing-subscriber/src/layer.rs

```diff
@@ -994,6 +994,29 @@ where
         }
     }
 
+    /// Finds the [`SpanRef`] associated with an [`Event`], if it has one.
+    ///
```

Comment:
how about something like:
```suggestion
    /// Returns a [`SpanRef`] for the parent span of the given [`Event`], if
    /// it has a parent.
    ///
    /// If the event has an explicitly overridden parent, this method returns
    /// a reference to that span. If the event's parent is the current span,
    /// this returns a reference to the current span, if there is one. If this
    /// returns `None`, then either the event's parent was explicitly set to
    /// `None`, or the event's parent was defined contextually, but no span
    /// is currently entered.
    ///
```